stable/admin/templates/ingress.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.admin.ingress }}
+{{- if eq (include "defaultfalse" .Values.admin.ingress.enabled) "true" }}
+{{- if .Values.admin.ingress.api }}
+{{- if .Values.admin.ingress.api.hostname }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ template "admin.fullname" . }}-api
+  annotations:
+    {{- toYaml .Values.admin.ingress.api.annotations | trim | nindent 4 }}
+  labels:
+    app: {{ template "admin.fullname" . }}
+    chart: {{ template "admin.chart" . }}
+    release: {{ .Release.Name | quote }}
+    domain: {{ .Values.admin.domain }}
+    group: nuodb
+spec:
+  {{- if .Values.admin.ingress.api.tls }}
+  tls:
+    - hosts:
+      - {{ .Values.admin.ingress.api.hostname | quote }}
+    {{- if .Values.admin.ingress.api.secretName }}
+      secretName: {{ .Values.admin.ingress.api.secretName }}
+    {{- end -}}
+  {{- end }}
+  {{- if .Values.admin.ingress.api.className }}
+  ingressClassName: {{ .Values.admin.ingress.api.className | quote }}
+  {{- end }}
+  rules:
+  - host: {{ .Values.admin.ingress.api.hostname | quote }}
+    http:
+      paths:
+      - path: {{ default "/" .Values.admin.ingress.api.path }}
+        pathType: Prefix
+        backend:
+          service:
+            name: {{ .Values.admin.domain }}-{{ .Values.admin.serviceSuffix.clusterip }}
+            port:
+              name: 8888-tcp
+{{- end }}
+{{- end }}
+{{- if .Values.admin.ingress.sql }}
+{{- if .Values.admin.ingress.sql.hostname }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ template "admin.fullname" . }}
+  annotations:
+    {{- toYaml .Values.admin.ingress.sql.annotations | trim | nindent 4 }}
+  labels:
+    app: {{ template "admin.fullname" . }}
+    chart: {{ template "admin.chart" . }}
+    release: {{ .Release.Name | quote }}
+    domain: {{ .Values.admin.domain }}
+    group: nuodb
+spec:
+  {{- if .Values.admin.ingress.sql.tls }}
+  tls:
+    - hosts:
+      - {{ .Values.admin.ingress.sql.hostname | quote }}
+    {{- if .Values.admin.ingress.sql.secretName }}
+      secretName: {{ .Values.admin.ingress.sql.secretName }}
+    {{- end -}}
+  {{- end }}
+  {{- if .Values.admin.ingress.sql.className }}
+  ingressClassName: {{ .Values.admin.ingress.sql.className | quote }}
+  {{- end }}
+  rules:
+  - host: {{ .Values.admin.ingress.sql.hostname | quote }}
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: {{ .Values.admin.domain }}-{{ .Values.admin.serviceSuffix.clusterip }}
+            port:
+              name: 48004-tcp
```

Comment:
Do you know if this can be omitted? It does not seem to make any sense since we are actually not exposing a REST service on port 48004, but maybe we need this to force the load balancer to correctly dispatch connections with `SNI={{ .Values.admin.ingress.sql.hostname}}` to service.

Reply:
We need to expose NuoAdmin port 48004 used by the SQL clients to connect to the admin. This section of the ingress resource is configuring the `backend` service. I don't know if we can commit the `path` and the `pathType` as normally they are not interpreted by the Ingress controller when SSL passthrough is activated.